courses/machine_learning/time_series_prediction/modeling_keras.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>define above:</p><p> </p><p><code></code><code></p><p>def _create_split(phase):</p><p>&nbsp;&nbsp;floor, ceiling = 0, 70</p><p>&nbsp;&nbsp;if phase == 'VALID':</p><p>&nbsp;&nbsp;&nbsp;&nbsp;floor, ceiling = 70, 85</p><p>&nbsp;&nbsp;elif phase == 'TEST':</p><p>&nbsp;&nbsp;&nbsp;&nbsp;floor, ceiling = 85, 100</p><p>&nbsp;&nbsp;return '''</p><p>&nbsp;&nbsp;AND MOD(ABS(FARM_FINGERPRINT(symbol)), EVERY_N * 100) &gt;= (EVERY_N * {0})</p><p>&nbsp;&nbsp;AND MOD(ABS(FARM_FINGERPRINT(symbol)), EVERY_N * 100) &lt;&nbsp;(EVERY_N * {1})</p><p>&nbsp;&nbsp;'''.format(floor, ceiling)</p><p></code><code></code></p><p> </p><p>replace this part with</p><p> </p><p><code></code><code></p><p>subsample = _create_split(phase)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>wdty?</p>